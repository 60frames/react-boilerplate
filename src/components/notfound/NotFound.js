import React from 'react';

function NotFound() {
    return (
        <div>404 Not Found</div>
    );
}

export default NotFound;
