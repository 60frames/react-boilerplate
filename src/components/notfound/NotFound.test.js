/* eslint-env jasmine */

import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from 'components/notfound/NotFound';

describe('components/notfound/NotFound', () => {

    it('renders correctly', () => {
        const component = renderer.create(
                <NotFound />
        );
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

});
