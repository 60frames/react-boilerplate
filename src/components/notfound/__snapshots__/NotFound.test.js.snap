exports[`components/notfound/NotFound renders correctly 1`] = `
<div>
  404 Not Found
</div>
`;
